mantidproject/mantid
# Fix bug with tiled plots

Tiled plots do not have the correct size when reloaded. 

**To test:**
  - Create a tiled plot by right clicking on a workspace, selecting some spectra and choosing "tile plot"
 - Save and reload the project
 - Tile plot should be shown correctly now

Fixes #18199 .

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
